pkg/util/spill/agg_spill_util.go

```diff
@@ -0,0 +1,416 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spill
```

Comment:
We can rename the filename. The functions in this file can be used by others except agg.
Maybe we can rename it to `serialization_util.go`.
And the package name can be renamed to `serialization`